src/frontend/components/SøknadsSteg/VelgBarn/LeggTilBarn/NyttBarnKort.test.tsx

```diff
@@ -119,12 +119,14 @@ test(`Kan legge til barn`, async () => {
                         åpen.push(1);
                     }}
                 />
-                <LeggTilBarnModal
-                    erÅpen={åpen.length > 0}
-                    toggleModal={() => {
-                        åpen.pop();
-                    }}
-                />
+                {åpen.length > 0 && (
+                    <LeggTilBarnModal
+                        erÅpen={åpen.length > 0}
+                        lukkModal={() => {
+                            åpen.pop();
+                        }}
+                    />
+                )}
```

Comment:
Denne testen er jeg ikke sikker på om gir så mye verdi lenger. Deler av testen som går på åpning/lukking av modal er hvertfall litt bortkastet. Pga ny modal som rendres fra start må man legge inn sjekken `eråpen &&` før modalen. Linje 134 i denne testen tester egentlig kun derfor at åpen.length = 0, siden modal ikke rendres. Vil du ta en titt på testen, @daphneleebeek og se hva du syns?

Reply:
Ja, flyten for åpne og lukke er kanskje ikke så relevant lenger 😊 kan være fint å beholde det andre da, der hvor den testen hva som skal til for at knappen blir primary?

Reply:
Enig! Fjerner deler av testen, men beholder det aller meste